src/Silex/EventListener/MiddlewareListener.php

```diff
@@ -37,12 +39,8 @@ public function __construct(Application $app)
         $this->app = $app;
     }
 
-    /**
-     * Runs before filters.
-     *
-     * @param GetResponseEvent $event The event to handle
-     */
-    public function onKernelRequest(GetResponseEvent $event)
+
+    public function onKernelRequest(RequestEvent $event)
```

Comment:
From: https://github.com/symfony/http-kernel/blob/v4.4.50/Event/GetResponseEvent.php

```
/**
 * @deprecated since Symfony 4.3, use RequestEvent instead
 */
```